conans/client/tools/scm.py

```diff
@@ -89,23 +99,34 @@ def run(self, command):
         command = self._configure_ssl_verify + command
         return super(Git, self).run(command)
 
-    def clone(self, url, branch=None, args=""):
+    def clone(self, url, element=None, args="", shallow=False):
```

Comment:
Even being experimental I prefer to not change the "branch" argument: https://docs.conan.io/en/latest/reference/tools.html#tools-git

Reply:
Document in the docstring that branch can be anything. Maybe a FIXME: for conan 2.0

Reply:
okay

Reply:
changed back to `branch`